File: 58. Length of Last Word/solution.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        if len(s) == 0:
            return 0
        
        lis = s.split(" ")
        index = len(lis)-1
        while len(lis[index]) == 0:
            index -= 1
            if index == -1:
                return 0
        return len(lis[index])

File: 58. Length of Last Word/solution.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int res = 0, i = s.length() - 1;
        bool start = false;
        while(i >= 0){
            if(s[i] != ' '){
                start = true;
                res ++;
            }
            else if(s[i] == ' ' && start){
                break;
            }
            i--;
        }
        return res;
    }
};
